### DISCUSSION TITLE
Strange lines appear when using visit to visualize

### DISCUSSION THREAD

**Discussion by tianninglyu** [PrincetonUniversity/athena#500] [Q&A]
I use the problem generator jet.cpp and the input file athinput.jet, and visualize the data with visIt. When I use vtk to output, everything is correct. However, when I turn to use hdf5 output, strange lines appear. Here is the image.
![vtk](https://github.com/PrincetonUniversity/athena/assets/130236675/847dcf6b-4aa5-4f7d-9e3a-4af943cc0397)
![h5](https://github.com/PrincetonUniversity/athena/assets/130236675/aa1d7d18-7c8d-4ed2-ab06-415ceb0583a1)
I have three conjectures: 1. it is due to the output format; 2. It is due to parallel computing (HDF5 output files are obtained by parallel computation while vtk output files are not); 3. It is due to the visIt software.
I wonder if you have some experience about that. 
Thanks!

**Comment by felker** [comment]
Have you tried 1) plotting the HDF5 files in matplotlib or another alternative? and 2) running without MPI?

What does your `<output>` parameter block look like for HDF5?

**Reply by tianninglyu** [reply]
I have tried to visualize the hdf5 data with the yt python package, and everything goes well.

I have tried running without MPI, but when without MPI, I use vtk output but not hdf5. In this case, the data can be visualized correctly with visIt.

The output is as follows:
```
output1
file_type  = hst      # History data dump
dt         = 0.1      # time increment between outputs

output2
file_type  = hdf5      # Binary data dump
variable   = prim     # variables to be output
dt         = 1.0      # time increment between outputs

```

Best wishes and thank you for your consideration.
